packages/malloy-render/src/component/table.ts

```diff
@@ -86,19 +86,23 @@ const renderFieldContent = (
       .rowLimit=${options.pinnedHeader ? 1 : Infinity}
     ></malloy-table>`;
   }
-  if (options.pinnedHeader)
-    return renderCell(f, '', {
-      hideStartGutter: isFirstChild(f),
-      hideEndGutter: isLastChild(f),
-    });
-  return renderCell(f, row.cell(f).value, {
+  let value = row.cell(f).value;
+  if (options.pinnedHeader) value = '';
+  if (f.isAtomicField() && f.isNumber())
```

Comment:
Should this be an `else if`? Is it possible for `options.pinnedHeader` and `f.isAtomicField() && f.isNumber()`, in which case you'd be asserting `'' is number` (which still works because string has a `toLocaleString()` method, but might eventually have other side effects?)